earth_enterprise/src/fusion/tools/gee_test/README

```diff
@@ -25,6 +25,7 @@ sudo /opt/google/bin/geconfigureassetroot --new --assetroot  /usr/local/google/g
 
 sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test/assets
```

Comment:
I'm not sure what you mean here? /opt/google/bin contains the executables and /usr/local/google contains the asset root. Should I add that to a comment?